runbotics/runbotics-orchestrator-ui/src/src-app/components/Tile/ProcessTile/ProcessTile.actions.tsx

```diff
@@ -73,7 +77,10 @@ const ProcessTileActions: VFC<ProcessTileProps> = ({ process }) => {
                 </IconButton>
                 <Menu id="simple-menu" anchorEl={anchorEl} keepMounted open={!!anchorEl} onClose={handleClose}>
                     <If condition={hasEditProcessAccess}>
-                        <MenuItem onClick={() => setShowDialog(true)}>
+                        <MenuItem onClick={() => {
+                            setShowDialog(true);
+                            handleClose();
+                        }}>
```

Comment:
Just a suggestion, what do you think about moving this to something like handleMenuItemClick, imo it would improve readability. You might also think about renaming the current handleClick function.